CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Mattermost Push Proxy Changelog
+
+## 5.8 Release
+- Release Date: March 20, 2019
+- Server Versions Supported: Server v4.10+ is required XXXXXX dev confirm how far back we support this
```

Comment:
@elias please help confirm here

Reply:
since we added push notifications, so I don't think this is needed at all.